packages/notification-center/src/components/notification-center-web-component/notification-center-content-component.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
The Notification Center Content component without the popover logic.
This component is converted to the Web Component.